packages/mdc-textfield/_mixins.scss

```diff
@@ -62,3 +62,263 @@
     }
   }
 }
+
```

Comment:
All of this code was moved from `/mdc-textfield/mdc-text-field.scss`. I made sure not to alter actual CSS. Pretty much a copy/paste, and extracted the state selectors (`--focused`, `--disabled`, etc)